src/js/views/admin/list/clinicians-all_views.js

```diff
@@ -100,6 +57,13 @@ const ItemView = View.extend({
   onRender() {
     this.showRole();
   },
+  onClick() {
+    if (this.model.isNew()) {
+      Radio.trigger('event-router', 'clinician:new');
```

Comment:
I think this will cancel anything not saved in the sidebar.  Best probably to just `return;` here?